### PR TITLE
{system}[intel/2015a] OpenSSL 1.0.1m (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
@wpoely86: please review?

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b34037b35611973415f7 for a full test report.

**Comment by wpoely86** [comment]
looks fine

File: easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1m-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.openssl.org/source/']
+
+patches = [
+    'OpenSSL-%(version)s_icc-fixes.patch',
+    'OpenSSL-%(version)s-fix_parallel_build-1.patch',
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1m-intel-2015a.eb]: drop `-1`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/39b192858ccf51bd41b0 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/134e2a348ae75ec1c34e for a full test report.

**Comment by boegel** [comment]
Thanks for the review/test @wpoely86!

**Pull request merged**